plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/Completion/Types.hs

```diff
@@ -39,7 +39,7 @@ instance Pretty Log where
 
 type instance RuleResult ParseCabal = Parse.GenericPackageDescription
 
-data ParseCabal = ParseCabal
+data ParseCabal = GetCabalDiagnostics
```

Comment:
rename the type constructor too? Also the old name was somewhat plausible, right, since it does produce the GPD. So it's a bit like Typecheck: produces the typechecked module and diagnostics. Unsure.

Reply:
Good call, I had too much trust in the rename feature to also read my mind :see_no_evil: